### PR TITLE
Include links of imprints into link replacement

### PR DESCRIPTION
**Pull request by MizukiTemma**
### Short description

This PR adds `ImprintPageTranslation` into the target models of `replace_links` so links in imprints can be replaced too by the "Search&Replace" on the uppr right corner of the broken link list.

### Proposed changes

- Add `ImprintPageTranslation` into the list `models` in the function `replace_links`

### Side effects

- Should we probaly think of centralised check for such process of adding/removing a model to/from our link features? Somthing like, listing models in one single list (as an environment variable, for exmaple) and we add a check whether all models of the list are included at the points where you need to add changes while removing/adding a model to the link check feature.

### Resolved issues

Fixes: #2982

__________________________________________________

[Pull Request Review Guidelines](https://digitalfabrik.github.io/integreat-cms/pull-request-review-guide.html)

### CODE REVIEW

File: integreat_cms/release_notes/current/unreleased/2982.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+en: Include links in implints in link replacement
+de: Beziehe Links in Implikationen in Link-Ersetzung ein
```
Comment (david-venhoff) [inline integreat_cms/release_notes/current/unreleased/2982.yml:2]:
```suggestion
en: Include links in imprints for link replacement
de: Beziehe Links in Impressen in der Link-Ersetzung ein
```

**Review by david-venhoff — APPROVED** [review]
Thanks!

> Should we probaly think of centralised check for such process of adding/removing a model to/from our link features? Somthing like, listing models in one single list (as an environment variable, for exmaple) and we add a check whether all models of the list are included at the points where you need to add changes while removing/adding a model to the link check feature.

I think this is a good idea. For this case though I think it would be best to just completely rewrite the `replace_links` function. Over time, it has become quite messy and now we have the `get_region_links` function that we could use instead to make `replace_links` both faster and cleaner.
Do you want to do that in this pr? Otherwise I could also do that in a follow-up pr.

**Comment by MizukiTemma** [comment]
@david-venhoff Thank you for review and idea 😸 I'd suggest to do it in another PR to stay in the policy "one PR, one purpose".

**Review by deen13 — APPROVED** [review]
👍 😄

**Pull request merged**